test/test_translator.py

```diff
@@ -1,17 +1,13 @@
-import sys
-sys.path.append("..")
+
 from translator.source_code import icone_translator
 import xmltodict
 import json
-from jsonschema import validate
-from datetime import datetime
-import subprocess
-
+import jsonschema
 
 
 
 
-#Unit testing code
+#Unit testing code for icone_translator.py
 def test_parse_incident() :
```

Comment:
Can we break these unit tests out to test a single unit each? This one seems to be testing a success case, and a case where we don't find a direction. Both are valid tests, and should be tested individually.

There's several additional edge cases to test with this `parse_incident` method as well, for instance what happens when we pass `None` to the method? Or invalid data?

Reply:
added additional edge cases with valid test function name and pushed into dev.